docs/samples/bentoml/README.md

```diff
@@ -0,0 +1,189 @@
+# Predict on an InferenceService using BentoML
+
+This guide demonstrates how to serve a scikit-learn based iris classifier model with BentoML
+and deploying the BentoML model server with KFServing. The same deployment
+steps are also applicable for models trained with other machine learning frameworks, see
+more BentoML examples [here](https://docs.bentoml.org/en/latest/examples.html).
+
+[BentoML](https://bentoml.org) is an open-source platform for high-performance ML model
+serving. It makes building production API endpoint for your ML model easy and supports all
+major machine learning training frameworks, including Tensorflow, Keras, PyTorch, XGBoost,
+scikit-learn and etc.
+
+BentoML comes with a high-performance API model server with adaptive micro-batching support,
```

Comment:
@yubozhao I am interested in micro-batching support, is there any doc you can point me?

Reply:
Hi @yuzisun here’s the doc about how to enable micro batching in BentoML: https://docs.bentoml.org/en/latest/concepts.html#adaptive-micro-batching

We are still working on a more detailed design overview document for it, which will explain the overall design and some key design decisions we made when building it. That doc will be out soon!